packages/cloud-cognitive/src/components/Datagrid/Datagrid/DatagridVirtualBody.js

```diff
@@ -58,7 +75,14 @@ const DatagridVirtualBody = (datagridState) => {
   const visibleRows = (DatagridPagination && page) || rows;
 
   return (
-    <TableBody {...getTableBodyProps()} onScroll={onScroll}>
+    <>
+    <div
+      className={`${blockClass}__head-warp`}
+      style={{ width: gridElement?.clientWidth, overflow: 'hidden' }}
```

Comment:
would move to ref and use `gridElement.current`

Reply:
Updated to use `useRef`